tests/integration/test_postgresql_protocol/test.py

```diff
@@ -162,6 +163,41 @@ def test_python_client(started_cluster):
     cur.execute("DROP DATABASE x")
 
 
+def test_copy_command(started_cluster):
+    node = cluster.instances["node"]
+
+    ch = py_psql.connect(
+        host=node.ip_address,
+        port=server_port,
+        user="default",
+        password="123",
+        database="",
+    )
+    cur = ch.cursor()
+    file_index = random.randint(0, 100000000)
+
+    cur.execute("drop table if exists test;")
+    cur.execute("drop table if exists test_recreated;")
+
+    cur.execute("create table test (x UInt32) engine=Memory();")
+    cur.execute("insert into test values (42);")
+    cur.execute("insert into test values (43);")
+
+    cur.execute(f"copy test to file('test{file_index}.csv', 'CSV', 'x UInt32');")
```

Comment:
@scanhex12 Maybe I miss something but I assume you worked on this PR in response to #71175 "Completeness of the implementation of PostgreSQL wire protocol" which says "and COPY command into the PostgreSQL compatibility protocol in ClickHouse". What it means is an optimization for the Postresql _wire protocol_, see [here](https://www.dolthub.com/blog/2024-09-17-tabular-data-imports/). This PR adds a new COPY command but it doesn't change anything in the Postgresql wire protocol.

(Btw., any new SQL statement would need docs and a lot more tests, testing all edge cases systematically.)

Reply:
Sorry for the misunderstanding, I fixed the postgres protocol